<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WMessages</class>
 <widget class="QWidget" name="WMessages">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>526</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Messages</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="1,0,0,0,0,2,0">
   <item>
    <widget class="QTableView" name="tblMessages"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnAdd">
       <property name="text">
        <string>Add</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnDelete">
       <property name="text">
        <string>Delete</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="lblSubject">
     <property name="text">
      <string>Subject:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="txtSubject">
     <property name="text">
      <string/>
     </property>
     <property name="placeholderText">
      <string>Subject</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="lblMessage">
     <property name="text">
      <string>Message:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPlainTextEdit" name="txtMessage"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnSave">
       <property name="text">
        <string>Save</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
